lua/scrollbar/handlers/search.lua

```diff
@@ -18,6 +18,12 @@ function M.search(cb)
 
     local command = config.search.command
 
+    if vim.fn.executable(command) ~= 1 then
+        vim.notify(command .. " was not found on your path, trying to fall back to grep", vim.log.levels.ERROR)
+        command = "grep"
+    end
+
+
     if vim.fn.executable(command) ~= 1 then
```

Comment:
This probably isn't needed anymore

Reply:
Probably worth just keeping this just in case

Reply:
Yes, this basically cleanly saveguards against not having hero installed. I know grep is on every Linux system, I don't know about macOS, and I'm sure Windows doesn't have it.

Reply:
But this check on line `27` and on line `21` are the same?

Reply:
I believe it's to check for `rg` then `grep`